<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWnd</class>
 <widget class="QMainWindow" name="MainWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCBSD Control Panel</string>
  </property>
  <property name="windowIcon">
   <iconset resource="controlpanel.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="toolButton_2">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="controlpanel.qrc">
          <normaloff>:/images/images/refresh.png</normaloff>:/images/images/refresh.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="DEChooserButton">
        <property name="contextMenuPolicy">
         <enum>Qt::ActionsContextMenu</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="controlpanel.qrc">
          <normaloff>:/images/images/kde.png</normaloff>:/images/images/kde.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="deLaunchConfigApp">
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="placeholderText">
         <string>What do you want to configure?</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>579</width>
         <height>332</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="Line" name="SoftwareLine">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="SoftwareGBox">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Software and updates</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="SystemLine">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="SystemGBox">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>System management</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="HardwareLine">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="HardwareGBox">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Hardware</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="NetworkingLine">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="NetworkingGBox">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Networking</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="DELine">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="DEGBox">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Desktop environment</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="ToolsLine">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="ToolsGBox">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Tools</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAll">
   <property name="icon">
    <iconset resource="controlpanel.qrc">
     <normaloff>:/images/images/all_desktops.png</normaloff>:/images/images/all_desktops.png</iconset>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </action>
  <action name="actionKDE">
   <property name="icon">
    <iconset resource="controlpanel.qrc">
     <normaloff>:/images/images/kde.png</normaloff>:/images/images/kde.png</iconset>
   </property>
   <property name="text">
    <string>KDE</string>
   </property>
  </action>
  <action name="actionLXDE">
   <property name="icon">
    <iconset resource="controlpanel.qrc">
     <normaloff>:/images/images/lxde.png</normaloff>:/images/images/lxde.png</iconset>
   </property>
   <property name="text">
    <string>LXDE</string>
   </property>
  </action>
  <action name="actionGnome">
   <property name="icon">
    <iconset resource="controlpanel.qrc">
     <normaloff>:/images/images/gnome.png</normaloff>:/images/images/gnome.png</iconset>
   </property>
   <property name="text">
    <string>Gnome</string>
   </property>
  </action>
  <action name="actionEnlightenment">
   <property name="icon">
    <iconset resource="controlpanel.qrc">
     <normaloff>:/images/images/e16.png</normaloff>:/images/images/e16.png</iconset>
   </property>
   <property name="text">
    <string>Enlightenment</string>
   </property>
  </action>
  <action name="actionXFCE">
   <property name="icon">
    <iconset resource="controlpanel.qrc">
     <normaloff>:/images/images/xfce.png</normaloff>:/images/images/xfce.png</iconset>
   </property>
   <property name="text">
    <string>XFCE</string>
   </property>
  </action>
  <action name="actionUnsupported">
   <property name="icon">
    <iconset resource="controlpanel.qrc">
     <normaloff>:/images/images/unsupported_de.png</normaloff>:/images/images/unsupported_de.png</iconset>
   </property>
   <property name="text">
    <string>unsupported</string>
   </property>
  </action>
  <action name="actionMate">
   <property name="icon">
    <iconset>
     <normalon>:/images/images/mate.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Mate</string>
   </property>
   <property name="toolTip">
    <string>Mate</string>
   </property>
  </action>
  <action name="actionCinnamon">
   <property name="icon">
    <iconset>
     <normalon>:/images/images/cinnamon.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Cinnamon</string>
   </property>
   <property name="toolTip">
    <string>Cinnamon</string>
   </property>
  </action>
  <action name="actionSystemOnly">
   <property name="icon">
    <iconset resource="controlpanel.qrc">
     <normaloff>:/images/images/system_only.png</normaloff>:/images/images/system_only.png</iconset>
   </property>
   <property name="text">
    <string>System only items</string>
   </property>
   <property name="toolTip">
    <string>Hide all desktop items</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="controlpanel.qrc"/>
 </resources>
 <connections/>
</ui>
